database/buildSchema/04_organisation.sql

```diff
@@ -3,6 +3,7 @@
 CREATE TABLE public.organisation (
     id serial primary key,
     name varchar UNIQUE,
-    licence_number varchar,
-    address varchar
+    registration varchar UNIQUE,
+    address varchar,
+    logo_url varchar
```

Comment:
You've obviously merged master into here recently, so you'll need to also merge master into the base branch `#306-prepare-back-end-build ` to this doesn't show in your diff